grype/presenter/models/document.go

```diff
@@ -76,7 +77,7 @@ func NewDocument(id clio.Identification, packages []pkg.Package, context pkg.Con
 		Matches:        findings,
 		IgnoredMatches: ignoredMatchModels,
 		Source:         src,
-		Distro:         newDistribution(context.Distro),
+		Distro:         newDistribution(context, selectDistro(packages)),
```

Comment:
need to include channel here https://github.com/anchore/grype/blob/6f8284b6a8b4b01766b29f2bcc32a0f94ff4cdfa/grype/presenter/models/distribution.go#L9-L13